modules/gapi/include/opencv2/gapi/streaming/gstreamer/gstreamerpipeline.hpp

```diff
@@ -19,12 +19,12 @@ namespace gapi {
 namespace wip {
 namespace gst {
 
-class GAPI_EXPORTS GStreamerPipeline
+class GAPI_EXPORTS_W GStreamerPipeline
 {
 public:
     class Priv;
 
-    explicit GStreamerPipeline(const std::string& pipeline);
+    CV_WRAP explicit GStreamerPipeline(const std::string& pipeline);
```

Comment:
`CV_WRAP` -> `GAPI_WRAP`
Not sure that it's needed to wrap this `ctor` since `GStreamerPipeline` isn't supposed to be used, instead of this we've wrapped `make_gst_src`

Ok, let it be

Reply:
Actually GStreamerPipeline is as essential as GStreamerSource because you cannot create multi-source Gst-pipelines without it